### ISSUE TITLE
Show only 1 chat timestamp

### ISSUE THREAD

**Issue by HiImFlo** [HasKha/GWToolboxpp#421]
When "Show chat timestamps" is enabled in the native game settings and in toolbox, 2 chat timestamps will be shown.

In case both options are selected, show only the toolbox one and hide the other. This allows you to play without any tools injected and with toolbox without adjusting your settings.

**Comment by 3vcloud** [comment]
Fixed in 0516451e99c30216d6fe4b2f4923b55717d6f3d5

**Comment by Bu11etmagnet** [comment]
Does this mean that next time, there will be no timestamps until Toolbox is loaded? That's not very polite (changing the user's settings). It would be nicer to at least have the option to switch off TB's timestamp if the game's timestamps are turned on.

**Comment by 3vcloud** [comment]
Theres a bug with the way TB draws timestamps vs the in-game method - if I did this the other way around i.e. disabled TB timestamps when the user toggles native back on, you'd end up with 2 sets of timestamps before toolbox turns them off.

The easiest way to solve this issue was to "block" the setting change before it happened and show an error message rather than try to adjust the hook to cater for the native change

**Comment by Bu11etmagnet** [comment]
Does TB (with its own timestamps turned on) switch off the game's timestamps when it loads? I think that's bad behaviour (making persistent changes to the user's configuration without the user explicitly asking for it). It will leave the user wondering where the timestamps went, next time when GW is started but before TB is loaded (which is a short time for most TB users:)

**Comment by 3vcloud** [comment]
Toolbox timestamps are off by default, but if the user has previously enabled TB timestamps, the in-game timestamps will be disabled on load. 

I haven't added the code to re-enable timestamps (if they were enabled in-game) on close because that raises issues where you'd have to wait until the process is complete before closing toolbox and might cause crashing.

TLDR; we're very naughty
